start.sh

```diff
@@ -52,6 +52,9 @@ readonly BYTEMAN=$(findLib "org/jboss/byteman/byteman-install" "byteman-install-
 readonly CPLC_API=$(findLib "io/github/mkoncek/classpathless-compiler-api" "classpathless-compiler-api.*\.jar")
 readonly CPLC_UTIL=$(findLib "io/github/mkoncek/classpathless-compiler-util" "classpathless-compiler-util.*\.jar")
 readonly CPLC=$(findLib "io/github/mkoncek/classpathless-compiler" "classpathless-compiler.*\.jar")
+readonly BATIK_ALL=$(findLib "org/apache/xmlgraphics" "batik-all-.*\.jar")
+readonly XML_APIS=$(findLib "xml-apis/xml-apis-ext" "xml-apis-ext-.*\.jar")
+readonly XMLGRAPHICS=$(findLib "org/apache/xmlgraphics/xmlgraphics-commons" "xmlgraphics-commons-.*\.jar")
```

Comment:
I recently streamlined the library additions in .bat scripts. Would you mind adding these there as well?